extensions/3rdparty/ArrayExtension/ArrayExtension.php

```diff
@@ -547,7 +547,7 @@ function arrayintersect( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
 *    similar to arraymerge, this union works on values.
 */
 
-    function arrayunion( &$parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
+    function arrayunion( $parser, $arrayid_new , $arrayid1 , $arrayid2 ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)